minerl/Malmo/Minecraft/gradlew

```diff
@@ -6,12 +6,30 @@
 ##
 ##############################################################################
 
-# Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
```

Comment:
Why has this statement moved down? I assumed it was near the top of the file such that you would quickly see it you wanted to add custom options.